specification/trafficmanager/resource-manager/Microsoft.Network/stable/2022-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1701 @@
+        },
+        "minChildEndpointsIPv6": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The minimum number of IPv6 (DNS record type AAAA) endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
+        },
+        "geoMapping": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of countries/regions mapped to this endpoint when using the 'Geographic' traffic routing method. Please consult Traffic Manager Geographic documentation for a full list of accepted values."
+        },
+        "subnets": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "first": {
+                "type": "string",
+                "description": "First address in the subnet."
+              },
+              "last": {
+                "type": "string",
+                "description": "Last address in the subnet."
+              },
+              "scope": {
+                "type": "integer",
+                "description": "Block size (number of leading bits in the subnet mask)."
+              }
+            },
+            "description": "Subnet first address, scope, and/or last address."
+          },
+          "description": "The list of subnets, IP addresses, and/or address ranges mapped to this endpoint when using the 'Subnet' traffic routing method. An empty list will match all ranges not covered by other endpoints."
+        },
+        "customHeaders": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "Header name."
+              },
+              "value": {
+                "type": "string",
+                "description": "Header value."
+              }
+            },
+            "description": "Custom header name and value."
+          },
+          "description": "List of custom headers."
+        },
+        "alwaysServe": {
+          "type": "string",
+          "description": "If Always Serve is enabled, probing for endpoint health will be disabled and endpoints will be included in the traffic routing method.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "AlwaysServe",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Class representing a Traffic Manager endpoint properties."
+    },
+    "HeatMapModel": {
+      "properties": {
+        "systemData": {
```

Comment:
>systemData [](http://example.com/codeflow?start=9&length=10)

I only see systemData related properties getting added but your title mentions that the alwaysserve flag is being added. Did you follow the process outlined here to generate the PR?


The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended)  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) :  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" tag once you have fixed this issue.

Reply:
Thanks! will update accordingly in new PR